### ISSUE TITLE
Redux part is not deployed to git repo

### ISSUE THREAD

**Issue by Akshanshkaushal** [Souvikparua/COMPLETE-Fullstack-ecommerce-Responsive-MERN-App#2]
Add the redux part in the github rep , creating problem in running localhost

**Comment by Akshanshkaushal** [comment]
That redux part would be in your local repo . You have to push that to github only.

**Comment by Souvikparua** [comment]
Can you make a redux part according you or I upload it here

**Comment by SAGARSINGH-1** [comment]
Hey @Souvikparua please provide the Redux code or structure to work in that issue so far.

![image](https://github.com/Souvikparua/COMPLETE-Fullstack-ecommerce-Responsive-MERN-App/assets/81561733/b34a31c4-16a4-42d2-9a4a-6a1acb621a87)

**Comment by Souvikparua** [comment]
Ok

**Comment by SharmaDhruv2511** [comment]
is this issue still open ? I want to work on it @Akshanshkaushal

**Comment by SAGARSINGH-1** [comment]
> is this issue still open ? I want to work on it @Akshanshkaushal 

Yes

**Comment by SharmaDhruv2511** [comment]
the folder is not there in the project module that's why it is showing error.

**Comment by SAGARSINGH-1** [comment]
> the folder is not there in the project module that's why it is showing error.

yeah we know that that why I asked for structure of Redux store or code file

**Comment by maitri-vv** [comment]
hey! @Souvikparua its been a month to this issue being raised, i am still stuck onto the same problem? is no one working on this ??
![image](https://github.com/Souvikparua/COMPLETE-Fullstack-ecommerce-Responsive-MERN-App/assets/87691594/9ade6aca-3bc6-45f5-86e2-38be26d81486)
